student-store-ui/src/components/ShoppingCart/ShoppingCart.jsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import "./ShoppingCart.css";
+
+const ShoppingCart = ({ isOpen, products, shoppingCart }) => {
+  const subtotal = shoppingCart
+    .reduce((prev, curr) => prev + curr.price * curr.quantity, 0)
+    .toFixed(2);
+
+  const open = (
+    <>
+      <h2 className="shopping-cart-title">Shopping Cart</h2>
+      {shoppingCart.length === 0 ? (
+        <p>No items to add yet. Start shopping now!</p>
```

Comment:
this is the same message as the close var, maybe they could have been the same var